code/src/components/Artists.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const getSeparator = (i, length) => {
+  if (i === length - 2) {
+    return ' & ';
+  } else if (i < length - 2) {
+    return ', ';
+  }
+  return;
+};
+
+const Artists = (props) => {
+  const { artistsArray } = props;
+
+  return (
+    <div className='all-artists-names'>
+      {artistsArray.map((artist, i) => {
+        return (
+          <a
+            key={artist.id}
+            href={artist.external_urls.spotify}
+            className='artist'
+          >
+            {artist.name}
+            {getSeparator(i, artistsArray.length)}
+          </a>
+        );
```

Comment:
Nice that you did it with javascript, the css solution I went for was a bit confusing. I know what this code means but maybe could have a comment for explanation for others to know if they ever look at the code in the future :)